docs/source/release/v5.0.0/mantidworkbench.rst

```diff
@@ -26,94 +33,127 @@ New
 	plotBin(ws ,1,error_bars=True)
 
 	#Multi line plots
-	plotSpectrum([ws ,ws2],1)
-	plotBin([ws ,ws2],1)
+	plotSpectrum([ws,ws2],1)
+	plotBin([ws,ws2],1)
 	plotSpectrum([ws ,ws2],[1,2,3])
 	plotBin([ws ,ws2],[1,2,3])
-	plotSpectrum(["ws ","ws2"],[1,2,3])
+	plotSpectrum(["ws","ws2"],[1,2,3])
 
 	#plotting with spectra numbers
 	plotSpectrum(ws,spectrum_nums=2,error_bars=True)
 	plotSpectrum(ws,spectrum_nums=[2,4,6])
 	plotSpectrum([ws,ws2],spectrum_nums=list(range(2,10)))
 
 	# add a curve to an existing plot
-	plot1 = plotSpectrum(ws ,1,error_bars=True)
-	plot1 = plotSpectrum(ws ,error_bars=True,window=plot1)
+	plot1 = plotSpectrum(ws,1,error_bars=True)
+	plot1 = plotSpectrum(ws,1,error_bars=True,window=plot1)
 
 	# clear an existing plot use that window to plot
-	plot2 = plotSpectrum(ws ,1,error_bars=True)
+	plot2 = plotSpectrum(ws,1,error_bars=True)
 	plot2 = plotSpectrum(ws2,1,error_bars=True,window=plot2, clearWindow = True)
 
 	# plot as points not lines
-	plotSpectrum(ws ,1,error_bars=True,type=1)
-	plotBin(ws ,1,type=1)
+	plotSpectrum(ws,1,error_bars=True,type=1)
+	plotBin(ws,1,type=1)
 
 	# plot as waterfall graphs
 	plotSpectrum(["ws","ws2"],[1,2,3],waterfall=True)
 
 
-Improvements
-############
-
 .. figure:: ../../images/Notification_error.png
    :class: screenshot
    :width: 600px
    :align: right
 
-- If you have ever found it hard to spot when errors appear in the Messages window, and perhaps miss them if there are lots of graphs on the screen, then you will like this.  We have added system notifications when Mantid enounters an error, and directs you to look at the Messages window for details.  You can enable or disable these notifications from the File->Settings window.
+   *Error Notification Service*
+
+Improvements
+############
+
+**General**
+- The way scripts are parsed and executed has been vastly improved, solving issues related to the progress arrow, blank lines after a colon and tabs in strings.
+- Progress reporting for scripts has been vastly improved and now reports at the line level.
+- Toggle Whitespace in the editor now shows line endings as well as spaces and tabs
```

Comment:
Instead of being separate bullet points, these show up as a single block of text with no line breaks.